src/IRac.cpp

```diff
@@ -1905,6 +1910,7 @@ namespace IRAcUtils {
 #if DECODE_COOLIX
       case decode_type_t::COOLIX: {
         IRCoolixAC ac(0);
+        ac.on();
```

Comment:
Why is this statement required? `setRaw()` should (in theory) generate the correct internal state for `setPower()` from `result->value`.
As I understand it (I don't have access to a "Coolix" compatible A/C system), every IR message that is NOT an "off" command is considered an "on" message. Is that a correct assumption?